### DISCUSSION TITLE
Welcome to ERMACK Discussions!

### DISCUSSION THREAD

**Discussion by aw350m33d** [Security-Experts-Community/ERMACK#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about ❓.
  * Share ideas 💡.
  * Engage with other community members 🤝.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by Pavel342** [comment]
Добрый день, очень заинтересовал проект. При изменении файла конфигурации на ru язык, появляется ошибка при сборке. Как ее можно исправить?
![image](https://github.com/Security-Experts-Community/ERMACK/assets/78688823/492f1eca-61f7-441e-a153-3c9e40acace5)

**Comment by aw350m33d** [comment]
Добрый день! 

Сегодня посмотрю почему эта ошибка возникает.

В целом, сейчас две локализации не синхронизированы и основные наработки находятся в английской локализации.

**Comment by aw350m33d** [comment]
> Добрый день, очень заинтересовал проект. При изменении файла конфигурации на ru язык, появляется ошибка при сборке. Как ее можно исправить? ![image](https://user-images.githubusercontent.com/78688823/246352460-492f1eca-61f7-441e-a153-3c9e40acace5.png)

Ошибка исправлена ([commit](https://github.com/Security-Experts-Community/ERMACK/commit/eb386caed564c0a324f08c9cdff31db67e9c3dae)). Дополнительно поправил кодировки некоторых файлов. 

Теперь русская версия собирается. Но ещё много работы по локализации действий.
